### PR TITLE
Update to Svc::Framer design and implementation

### PR DESCRIPTION
**Pull request by bocchino**
After design review and code review.

### CODE REVIEW

**Review by kevin-f-ortega — APPROVED** [review]
LGTM. We went through this together.

**Review by arizvi786 — APPROVED** [review]
@bocchino great work!

**Review by LeStarch — CHANGES_REQUESTED** [review]
One minor spelling issue

File: Svc/Framer/Framer.fpp [hunk 1]
```diff
@@ -1,25 +1,38 @@
 module Svc {
 
-  @ A component for framing unframed input
+  @ A component for deframing input for transmission to the ground
```
[the remark this answers is not in the document]
Comment (bocchino) [reply Svc/Framer/Framer.fpp]: Good catch, fixed!

**Review by LeStarch — APPROVED** [review]

**Comment by LeStarch** [comment]
## Pre-CI Checks: Must perform before launching CI
- [x] No obviously malicious code introduced
- [ x PR is against `devel`. Use `Edit -> change branch` to fix, or close PR.

## Code Review
- [x] Code follows flight software quality guidelines
- [x] Changes are of reasonable length for review

## Procedural Checks
- [x] PR passes CI
- [x] Accept with `squash-and-merge`

File: Svc/Framer/docs/sdd.md [hunk 2]
```diff
@@ -1,46 +1,317 @@
...
+<div>
+<img src="img/top/event.png" width=1000/>
+</div>
+
+The `eventLogger` instance sends event packets to the `framer` instance.
+
+**Topology 3: File packets:**
+
+<div>
+<img src="img/top/file.png" width=1000/>
```
Comment (jsoref) [inline Svc/Framer/docs/sdd.md:202]: Fwiw, afaict this file (`file.png`) is ambiguous and broke the autodocumentation (which flattens files into a single directory).

I've posted #1599 which renames it. (I can't hang a comment on an image file, so I'm hanging it here where it's being used.)

**Pull request merged**